ui/app/components/tools/wrap.js

```diff
@@ -24,15 +25,33 @@ export default class ToolsWrap extends Component {
   @tracked buttonDisabled = false;
   @tracked token = '';
   @tracked wrapTTL = null;
-  @tracked wrapData = '{\n}';
+  @tracked wrapData = null;
   @tracked errorMessage = '';
+  @tracked showJson = true;
+
+  get startingValue() {
+    // must pass the third param called "space" in JSON.stringify to structure object with whitespace
+    // otherwise the following codemirror modifier check will pass `this._editor.getValue() !== namedArgs.content` and _setValue will be called.
+    // the method _setValue moves the cursor to the beginning of the text field.
+    // the effect is that the cursor jumps after the first key input.
+    return JSON.stringify({ '': '' }, null, 2);
```

Comment:
I think the white space is a nice UX improvement! Even if users are likely just copying and pasting their data - it's nice to match other instances of this component.